Test/Main.hs

```diff
@@ -24,4 +25,42 @@ tests :: Spec
 tests = aroundAll (withIHPApp WebApplication config) do
     describe "Test" do
         it "should work" $ withContext do
-            True `shouldBe` True
\ No newline at end of file
+            True `shouldBe` True
+
+        itShouldAllowNewAndCreate ParagraphCta NewParagraphCtaAction CreateParagraphCtaAction
```

Comment:
If I remove the `newAction` and change this to 

`itShouldAllowNewAndCreate ParagraphCta CreateParagraphCtaAction`

Then the error I get is

<img width="883" alt="image" src="https://github.com/amitaibu/ihp-landing-page/assets/125707/55656586-3b04-4851-9d95-011505cdc883">